### PR TITLE
Added tests for the oculus integration tool

### PR DESCRIPTION
**Pull request by RogPodge**
## Overview
Added tests and cleaned up files associated with #8259 and #8386

### CODE REVIEW

**Review by wiwei — APPROVED** [review]
Change lgtm modulo @keveleigh's comment

File: Assets/MRTK/Tests/EditModeTests/Editor/OculusIntegrationSetupTests.cs [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+namespace Microsoft.MixedReality.Toolkit.Tests.EditModeTests.Editor
+{
+    /// <summary>
+    /// Set of tests to validate MRTK Toolbox Editor window.
+    /// </summary>
+    public class OculusIntegrationSetupTests
+    {
+        /// <summary>
+        /// Tests that the MixedRealityToolboxWindow can load without exception and that none of
```
Comment (MaxWang-MS) [inline Assets/MRTK/Tests/EditModeTests/Editor/OculusIntegrationSetupTests.cs:24]: Nit: copypasta

File: Assets/MRTK/Tests/EditModeTests/Editor/OculusIntegrationSetupTests.cs [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+            // Configure the oculus settings successfully
+            OculusXRSDKHandtrackingConfigurationChecker.ConfigureOculusIntegration();
+
+            // Check that the definitions and amdefs are set up correctly when Oculus Integration is present
+            Assert.IsTrue(OculusXRSDKHandtrackingConfigurationChecker.ReconcileOculusIntegrationDefine());
+            ValidateAsmdefs(true);
+
+            // Remove the dummy asset and clean up the project to prevent errors
+            AssetDatabase.DeleteAsset("Assets/OculusProjectConfig.asset");
```
[the remark this answers is not in the document]
Comment (keveleigh) [reply Assets/MRTK/Tests/EditModeTests/Editor/OculusIntegrationSetupTests.cs:44]: Can the creation / deletion steps be part of `Setup` and `TearDown` steps?

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/a09ae8a33836ef46cfad976bc08a7fc86cca45d3/Assets/MRTK/Tests/EditModeTests/Tools/ExtensionServiceCreatorTests.cs#L22-L24

**Pull request merged**